CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ### 0.11.0 (Major Release)
 
+#### Changes all models
```

Comment:
I think you've committed the changelog with the title stuff in as well - I deleted that blank 'changes all models' heading and moved the title section in a previous PR...